cloud4all.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="chrome=1" />
```

Comment:
I copied this from this page that Steve Githens created: https://github.com/GPII/gpii.github.io/blob/master/index.html. 
@sgithens Do we really need that meta tag?